schema/bom-1.6.xsd

```diff
@@ -77,11 +77,11 @@ limitations under the License.
                 A version range specified in Package URL Version Range syntax (vers) which is defined at https://github.com/package-url/purl-spec/VERSION-RANGE-SPEC.rst
 
                 Example values:
-                - "9.0.14"
-                - "1.2.3|>=2.0.0|<5.0.0"
-                - "0.0.0|0.0.1|0.0.2|0.0.3|1.0|2.0pre1"
-                - ">=1.0.0-beta1|<=1.7.5|>=7.0.0-M1|<=7.0.7|>=7.1.0|<=7.1.2|>=8.0.0-M1|<=8.0.1"
-                - ">=2.2.0|!= 2.2.1|<2.3.0"
+                - "vers:cargo/9.0.14"
+                - "vers:npm/1.2.3|>=2.0.0|<5.0.0"
+                - "vers:pypi/0.0.0|0.0.1|0.0.2|0.0.3|1.0|2.0pre1"
+                - "vers:tomee/>=1.0.0-beta1|<=1.7.5|>=7.0.0-M1|<=7.0.7|>=7.1.0|<=7.1.2|>=8.0.0-M1|<=8.0.1"
```

Comment:
For the vers to be correct, the ranges must also be [normalized](https://github.com/package-url/purl-spec/blob/version-range-spec/VERSION-RANGE-SPEC.rst#normalized-canonical-representation-and-validation) as explained here. For instance, let's take an incorrectly normalized vers where `<=8.0.1` is at the front.

```
vers:tomee/<=8.0.1|>=1.0.0-beta1|<=1.7.5|>=7.0.0-M1|<=7.0.7|>=7.1.0|<=7.1.2|>=8.0.0-M1
```

This would unfortunately match more versions than necessary. For now, we perhaps only add a warning since the intention here is to capture the range and not to use it for validation.